paper/paper.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Game Theory and Python: An educational tutorial to game theory and
+repeated games using Python"
+tags:
+- game theory
+- repeated games
+- prisoner's dilemma
+- axelrod python
+authors:
+    - name: Nikoleta E. Glynatsi
+    ORCID: 0000-0002-2943-3622
+    affiliation: 1 affiliations:
+    - name: Vincent Knight
+    ORCID: 0000-0002-4245-0638
+    affiliation: 1
+affiliations:
+    - name: Department of Mathematics, Cardiff University, Senghennydd Rd, Cardiff CF24 4AG
+    index: 1,
+date: some date in 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+These materials are an open source educational tutorial aimed at introducing
+participants to game theory and more specifically to repeated games. The
+tutorial uses the open source library called
+`Axelrod-Python` and [Jupyter
+Notebooks](https://jupyter.org) making this an open source, reproducible and
+interactive tutorial.
+
+# Statement of Need
+
+Game theory is a field of applied mathematics interested in strategic
```

Comment:
Before this, have:


```This tutorial provides an introduction to using Python in the context of Game Theory.
It could be used by individuals familiar with Python to start to learn Game Theory b
ut could also be used as a pathway to programming through the interesting
 subject that is Game Theory.
```

Reply:
Could you clarify please.

 Before `this` you mean `Game theory is a field of applied mathematics interested in ...`?

Is starting the Statement of Need with `It` a good idea?

Reply:
Yeah I meant an extra paragraph before the start but I agree that starting with `It` is not great, something like: `This tutorial...`.

Reply:
Fantastic, thank you!